src/utils/tests/object-inspector.js

```diff
@@ -221,4 +222,63 @@ describe("promises", () => {
     const node = getPromiseProperties(promise);
     expect(node.contents.value.type).to.eql("3");
   });
+
+  it("update actors when necessary", () => {
+    const item = {
```

Comment:
can we remove some of the data from this fixture so that it is much smaller. for instance, do we need the entire `value`? Can we remove `preview`'s contents?